CODE_OF_CONDUCT.rst

```diff
@@ -0,0 +1,154 @@
+   behavior are not acceptable. This includes, but is not limited to:
+   * Violent threats or violent language directed against another person
+   * Discriminatory jokes and language
+   * Posting sexually explicit or violent material
+   * Posting (or threatening to post) other people's personally identifying information ("doxing")
+   * Personal insults, especially those using racist or sexist terms
+   * Unwelcome sexual attention
+   * Advocating for, or encouraging, any of the above behavior
+   * Repeated harassment of others. In general, if someone asks you to stop, then stop
+
+6. **Moderate your expectations**. Please respect that community members choose
+   how they spend their time in the project. A thoughtful question about your
+   expectations is preferable to demands for another person's time.
+
+7. **When we disagree, try to understand why**. Disagreements, both social and
+   technical, happen all the time and Xonsh is no exception.  Try to
+   understand where others are coming from, as seeing a question from their
+   viewpoint may help find a new path forward.  And don’t forget that it is
+   human to err: blaming each other doesn’t get us anywhere, while we can learn
+   from mistakes to find better solutions.
+
+8. **A simple apology can go a long way**. It can often de-escalate a situation,
+   and telling someone that you are sorry is an act of empathy that doesn’t
+   automatically imply an admission of guilt.
+
+====================================
+Responding to inappropriate behavior
+====================================
+
+In some cases, individuals may violate the CoC in online or in-person situations.
+If any community member deems the action to be harmful to the community,
+they may act immediately to defuse the situation.
+This includes actions directed at the situation, rather than at an individual, such
+as locking a thread, *temporarily* suspending a user's account in the
+community forum, calling a meeting break, or generally acting to
+de-escalate the situation.
+
+If the incident involves physical danger, or involves a threat to anyone's safety
+(such as threats of violence), any member of the community may -- and should -- act
+unilaterally to protect the safety of any community member.
+This can include contacting law enforcement, other members of the Xonsh community,
+or other local personnel that are present.
+
+In situations where an individual community member acts unilaterally,
+they must inform the Code of Conduct committee as soon as possible,
+by emailing `xonduct@xon.sh <mailto:xonduct@xon.sh>`_.
+
+
+=========
+Reporting
+=========
+
+If you believe someone is violating the code of conduct, please report this in
+a timely manner. Code of conduct violations reduce the value of the community
+for everyone and we take them seriously.
+
+You can file a report by emailing `xonduct@xon.sh <mailto:xonduct@xon.sh>`_.
+Or by filling out this `form <form>`_
```

Comment:
Is it clear somewhere who the people are who can read CoC reports?

Reply:
Thanks for pointing this out @carmenbianca! I don't think it is clear.  My impulse is to say that it should be xore, but maybe that is too broad a group of people. Maybe there should be a subteam? Are there other 3rd parties we could report to? I am open to suggestions here.

Reply:
I know that in the FSFE we have a [CARE team](https://fsfe.org/about/codeofconduct.en.html#CARE) that is smaller than the core team (which is quite large). Also importantly, the president of the FSFE is not on the team.

I don't know what the correct choice for Xonsh is, though. But whatever the right choice is, listing somewhere the people who are in charge of CoC handling is a good idea :)

Reply:
This is definitely a TODO item before merging, I agree

Reply:
Thanks for raising this @carmenbianca !  I've already created a private team called @xonsh/xonduct (in the hopes of using it as a mailing list but github won't do that) but seeing your comment it makes very much sense now that it should be a publicly visible team, made up of some subset of xore (and potentially other members)